### DISCUSSION TITLE
plugins folder is not automatically created

### DISCUSSION THREAD

**Discussion by XiBeiChuiWang** [itzg/docker-minecraft-server#3059] [Q&A]
plugins folder is not automatically created，After manually creating the plugins folder, placing the plugins does not take effect. Confirm that the directory is successfully mounted.

docker compose：
services:
  mc:
    image: itzg/minecraft-server
    tty: true
    stdin_open: true
    ports:
      - "25565:25565"
    environment:
      EULA: "TRUE"
      MEMORY: "6G"
    volumes:
      - /root/mc/data:/data

**Comment by itzg** [comment]
It's not the responsibility of the image to create all of the subdirectories that are normally created by the server software. Besides, you're running a vanilla server type -- plugins aren't even supported. Perhaps you meant to use https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/server-types/paper/ which actually does use plugins.
